pkgs/by-name/re/recyclarr/package.nix

```
@@ -0,0 +1,78 @@
+{
+  lib,
+  openssl,
+  writeText,
+  git,
+  buildDotnetModule,
+  dotnetCorePackages,
+  fetchFromGitHub,
+  testers,
+}:
+let
+  nuget-config = writeText "nuget.config" ''
+    <configuration>
+      <packageSources>
+        <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+      </packageSources>
+    </configuration>
+  '';
+in
+buildDotnetModule (finalAttrs: rec {
```

```suggestion
buildDotnetModule (finalAttrs: {
```

I'd say we can use `finalAttrs.version` in the usages below without much issue